src/components/shared/Fields/InputStatus/InputStatus.tsx

```diff
@@ -33,7 +40,19 @@ const InputStatus = ({
   touched,
 }: Props) => {
   const { formatMessage } = useIntl();
-  const errorText = typeof error === 'object' ? formatMessage(error) : error;
+  const getErrorText = () => {
```

Comment:
Feels to me like this is doing the work of the `formatText` utility we have on `master`